src/rules/disallowed-tags.ts

```diff
@@ -0,0 +1,50 @@
+import { offOrStringArrayOrSeverityAndStringArray } from '../schemas'
+import Schema from '../schema'
+import Rule from '../rule'
+import { RawSchema, AcceptedSchema } from '../types'
+import Document from '../document'
+
+export default class DisallowedTags implements Rule {
+  public readonly name: string = 'disallowed-tags'
+
+  public readonly acceptedSchema: AcceptedSchema = offOrStringArrayOrSeverityAndStringArray
+
+  public readonly schema: Schema
+
+  public constructor(rawSchema: RawSchema) {
+    this.schema = new Schema(rawSchema)
+  }
+
+  public async run(document: Document): Promise<void> {
+    let DisallowedTags = this.schema.args as Array<string>
```

Comment:
Variables should start with a lower case letter

```suggestion
    let disallowedTags = this.schema.args as Array<string>
```